### DISCUSSION TITLE
export notes to a single file

### DISCUSSION THREAD

**Discussion by chenxizhang** [marp-team/marp#265] [Ideas] [answered]
I have an idea that want to export all the notes to a single file, so that I can print those notes. It's very useful if I just have one computer, like in a online meeting.

How do you think?

**Comment by yhatt** [comment]
Related: https://github.com/marp-team/marp-cli/issues/278

**Comment by yhatt** [comment, marked as answer]
[Marp CLI v1.7.0](https://github.com/marp-team/marp-cli/releases/tag/v1.7.0) has added `--notes` option to export presenter notes into a single text file.

```bash
# Export presenter notes to "your-slide.txt"
marp your-slide.md --notes
```

Give thanks a lot to @chrisns, a contributor of this feature.

**Reply by chenxizhang** [reply]
great
